packages/manager/apps/key-management-service/src/modules/secret-manager/pages/listing/listing.page.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { BaseLayout, Datagrid } from '@ovh-ux/manager-react-components';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useSecretList } from '@/modules/secret-manager/data/hooks/useSecretList';
+import {
+  DatagridCellPath,
+  DatagridCellVersion,
+  DatagridCreationDate,
+  DatagridItem,
+} from './ListingCells.component';
+
+type SecretListingParams = {
+  domainId: string;
+};
+
+export default function SecretListingPage() {
+  const { t } = useTranslation('secret-manager/common');
+  const { domainId } = useParams<SecretListingParams>();
+  const { data: secrets, isPending } = useSecretList(domainId);
+
+  const items: DatagridItem[] =
+    secrets?.map((secret) => ({
+      okmsId: domainId,
+      secret,
+    })) || [];
+
+  const columns = [
+    {
+      id: 'path',
+      cell: DatagridCellPath,
+      label: t('path'),
+    },
+    {
+      id: 'version',
+      cell: DatagridCellVersion,
+      label: t('version'),
+    },
+    {
+      id: 'createdAt',
+      cell: DatagridCreationDate,
+      label: t('creation_date'),
+    },
+  ];
+
+  return (
+    <BaseLayout header={{ title: t('secret_manager') }}>
+      <div className="flex flex-col gap-6">
+        <Datagrid
+          columns={columns}
+          items={items}
+          totalItems={items.length}
+          isLoading={isPending}
+          contentAlignLeft
+        />
+      </div>
```

Comment:
with only one child, this div with flex rules is not necessary.
```suggestion
        <Datagrid
          columns={columns}
          items={items}
          totalItems={items.length}
          isLoading={isPending}
          contentAlignLeft
        />
```

Reply:
yup, that was copy pasta I should have re-read